docker-compose.yml

```diff
@@ -60,6 +60,13 @@ services:
       << : *default-healthcheck
       test: "echo 'db.stats().ok' | mongo localhost:27017/test --quiet"
 
+  mongo-2.6:
+    image: mongo:2.6.12
+    command: ["--replSet", "router-mongo-replica-set"]
```

Comment:
Does this command need to reference router specifically? Just to check if this is a generic mongo 2.6 or a mongo 2.6 that can only be used by router?

Reply:
Good point. It is a generic mongo 2.6 but it's only being used by Router.

Reply:
Changed to remove the specific reference to `router`